packages/core-components/src/layout/HeaderTabs/HeaderTabs.tsx

```diff
@@ -17,6 +17,7 @@ import Box from '@material-ui/core/Box';
 import { makeStyles } from '@material-ui/core/styles';
 import TabUI, { TabProps } from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
```

Comment:
You are importing the same component here twice. `Tab` & `TabUI` both refer to the `default export` of `'@material-ui/core/Tab'`.